start.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Getting IPs
+source ${1:-hosts_prod.env}
+
+# Setting Grafana password
+sed -i "s/MONITORING_MASTER_PASSWORD/${MONITORING_MASTER_PASSWORD}/" grafana/config.monitoring
+
+# Setting IPs in different files
+sed -i "s/HOST_AND_IP_RONNY01/${RONNY01_IP}:${RONNY01_PORT}/" prometheus/http_hosts.yml
+sed -i "s/HOST_AND_IP_RONNY02/${RONNY02_IP}:${RONNY02_PORT}/" prometheus/http_hosts.yml
+sed -i "s/HOST_AND_IP_RONNY03/${RONNY03_IP}:${RONNY03_PORT}/" prometheus/http_hosts.yml
+sed -i "s/HOST_AND_IP_RONNY04/${RONNY04_IP}:${RONNY04_PORT}/" prometheus/http_hosts.yml
+sed -i "s/HOST_AND_IP_RONNY05/${RONNY05_IP}:${RONNY05_PORT}/" prometheus/http_hosts.yml
+sed -i "s/HOST_AND_IP_RONNY06/${RONNY06_IP}:${RONNY06_PORT}/" prometheus/http_hosts.yml
+sed -i "s/HOST_AND_IP_RONNY07/${RONNY07_IP}:${RONNY07_PORT}/" prometheus/http_hosts.yml
+sed -i "s/HOST_AND_IP_TELRAAM/${TELRAAM_IP}:${TELRAAM_PORT}/" prometheus/http_hosts.yml
```

Comment:
Theoretically this would `HOST_AND_PORT` or `IP_AND PORT`.  If we change this it should be reflected to all other variables that reference this as well.